tools/Install/setup.sh

```diff
@@ -105,9 +105,9 @@ get_platform() {
 show_help() {
   echo  'Usage: setup.sh <config-json-file> <xmos-tag> [OPTIONS]'
   echo  'The <config-json-file> can be downloaded from developer portal and must contain the following:'
-  echo  '   "clientId": "<OAuth client ID>"'
+  echo  '   "clientId": "<Auth client ID>"'
   echo  '   "productId": "<your product name for device>"'
-  echo  ' The  <xmos-tag> is the tag in the GIT repository xmos/avs-device-sdk'
+  echo  'The  <xmos-tag> is the tag in the GIT repository xmos/avs-device-sdk'
```

Comment:
Could we change this to something a little clearer ?  This tag is a version number. Could we make the usage : setup.sh <config-json-file> <avs-version-tag>